testing/trino-product-tests/src/main/java/io/trino/tests/hive/TestHiveViews.java

```diff
@@ -118,4 +118,22 @@ public void testLateralViewExplode()
                         row("nothing", null),
                         row("zero", null)));
     }
+
+    @Test(groups = HIVE_VIEWS)
+    public void testLateralViewJsonTupleAs()
+    {
+        onTrino().executeQuery("DROP TABLE IF EXISTS test_json_tuple_table");
+        onTrino().executeQuery("" +
+                "CREATE TABLE test_json_tuple_table AS " +
+                "SELECT 3 id, CAST('{\"user_id\": 1000, \"first.name\": \"Mateusz\", \"Last Name\": \"Gajewski\", \".NET\": true}' AS varchar) jsonstr");
+
+        onHive().executeQuery("DROP VIEW IF EXISTS test_json_tuple_view");
+        onHive().executeQuery("CREATE VIEW test_json_tuple_view AS " +
```

Comment:
looks good. Do you support the cN syntax alrady (without explicit column names)?

Reply:
No, it is not supported.
it fails here https://github.com/linkedin/coral/blob/d94cded024de24937397ae2499b011c0c096c715/coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/parsetree/ParseTreeBuilder.java#L205

Reply:
-> https://github.com/trinodb/trino/issues/7456